esp/src/src-react/components/forms/landing-zone/FileListForm.tsx

```diff
@@ -96,7 +96,7 @@ export const FileListForm: React.FunctionComponent<FileListFormProps> = ({
                     FileSpray.FileList({
                         request: {
                             Netaddr: machine,
-                            Path: data.path
+                            Path: data.path + pathSep
```

Comment:
There should probably be a check on `data.path` in case it already ends with a `pathSep`?

I think in bare metal this info comes from the config, so may be dependent on how the user entered it?